tweets/2021/04/kubernetes.tweet

```diff
@@ -0,0 +1 @@
+Yesterday our engineer Amim Knabben was co-author of an article about Defining Network Policy Conformance for CNI providers published in the @kubernetesio blog. Check this out: https://kubernetes.io/blog/2021/04/20/defining-networkpolicy-conformance-cni-providers/
```

Comment:
Make we can drop the "yesterday" word and post this tomorrow morning. It would reach a broader audience, I think. 

```suggestion
Our engineer Amim Knabben co-authored an article named "Defining Network Policy Conformance for CNI providers", published on the @kubernetesio blog. Check this out: https://kubernetes.io/blog/2021/04/20/defining-networkpolicy-conformance-cni-providers/
```